scripts/base_network.py

```diff
@@ -405,15 +441,22 @@ def _rebase_voltage_to_config(component):
 def base_network():
     buses = _load_buses_from_osm().reset_index()
     lines = _load_lines_from_osm(buses)
-    lines_dc = _load_links_from_osm(buses)
     transformers = _load_transformers_from_osm(buses)
+    converters = _load_converters_from_osm(buses)
+
+    lines_ac = lines[lines.tag_frequency.astype(float) != 0]
+    lines_dc = lines[lines.tag_frequency.astype(float) == 0]
+
+    # TODO Find a way to load a custom linetype. Currently the custom type definition seems to be ignored
```

Comment:
Should this todo remain?
If so please add an issue to track this point

Reply:
It seems that there is currently an option to load a custom line type. Removed this TODO